types/audit-details.d.ts

```diff
@@ -182,6 +180,8 @@ declare global {
         type: 'node';
         path?: string;
         selector?: string;
+        // TODO: can this be required? font-size and seo audits need to be updated.
```

Comment:
you might have already done this?

Reply:
don't have `boundingRect` for font-size gatherer.

just realized it should def. be optional. not all nodes are visible :)